tests/zipkin_test.py

```diff
@@ -215,6 +213,17 @@ def test_init_span_storage_wrong_type(self):
             ):
                 pass
 
+    def test_error_when_sample_rate_and_attrs_override(self):
+        with pytest.raises(ZipkinError):
```

Comment:
catch the exception with `as e` and then assert that the message contains `sample_rate should not be set when zipkin_attrs are provided`

Reply:
done